ee/query-service/main.go

```diff
@@ -148,7 +146,6 @@ func main() {
 		GatewayUrl:        gatewayUrl,
 		UseLogsNewSchema:  useLogsNewSchema,
 		UseTraceNewSchema: useTraceNewSchema,
-		UseLicensesV3:     useLicensesV3,
```

Comment:
we should remove all occurrences of this flag

Reply:
Yes missed this as well.